### PR TITLE
[feature/Junit-servlet5] test classes code commit

### PR DESCRIPTION
**Pull request by ShubhamAMU**

### CODE REVIEW

File: core/src/main/java/com/hcl/ecomm/core/servlets/MyAccountAddAddress.java [hunk 1]
```diff
@@ -63,11 +63,11 @@ protected void doPut(SlingHttpServletRequest request, SlingHttpServletResponse r
                     JSONObject customerAddress =  jsonPayload;
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/servlets/MyAccountAddAddress.java:70]: @ShubhamAMU : pl rename servlet too : MyAccountAddAddressServlet.java

Comment (ShubhamAMU) [reply core/src/main/java/com/hcl/ecomm/core/servlets/MyAccountAddAddress.java:70]: done

File: core/src/test/java/com/hcl/ecomm/core/servlets/MyAccountAddAddressTest.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
+package com.hcl.ecomm.core.servlets;
```
Comment (rishabh-behl) [inline core/src/test/java/com/hcl/ecomm/core/servlets/MyAccountAddAddressTest.java]: @ShubhamAMU : pl remane to MyAccountAddAddressServletTest

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@ShubhamAMU : pl add minor comments

File: core/src/test/java/com/hcl/ecomm/core/servlets/MyAccountAddAddressTest.java (hunk 2 above)
Comment (ShubhamAMU) [reply core/src/test/java/com/hcl/ecomm/core/servlets/MyAccountAddAddressTest.java]: @rishabh-behl  thats already done. Please check the files.

**Pull request merged**